source/MRMesh/MRMeshMetrics.h

```diff
@@ -95,8 +95,8 @@ MRMESH_API FillHoleMetric getUniversalMetric( const Mesh& mesh );
 /// This metric maximizes the minimal angle among all faces in the triangulation
 MRMESH_API FillHoleMetric getMinTriAngleMetric( const Mesh& mesh );
 
-// Very simple planar metric for only penalty for normal flip and area
-MRMESH_API FillHoleMetric getSimplePlanarMetric( const Mesh& mesh, EdgeId e0 );
+// Very simple area metric for only penalty for area and aspect ratio
```

Comment:
Better:
"This simple metric penalizes for large triangle area and large triangle aspect ratio"